src/components/ConfirmarionModal.vue

```diff
@@ -0,0 +1,95 @@
+ <template>
+  <section>
+    <div class="content-title">
+      <h4>Tem certeza</h4>
+      <h4>que deseja deletar este item?</h4>
+    </div>
+    <div class="content-btns">
+      <button @click="closeModal">Cancelar</button
+      ><button @click="deleteCar(id)">Sim</button>
+    </div>
+  </section>
+</template>
+ 
+ <script lang="ts">
+import { Vue, Component, Prop, Emit } from "vue-property-decorator";
+import carService from "@/services/carService";
+
+@Component
+export default class ConfirmationModal extends Vue {
+  @Prop() public id?: number;
+
+  @Emit("close-modal")
+  public closeModal() {
+    return this.closeModal;
+  }
+  @Emit("get-cars")
+  public getCars() {
+    return this.getCars;
+  }
```

Comment:
Por que você está retornando `this.getCars`? Olhando o código desse componente, essa variável não existe. Além disso, a função que é chamada pelo evento `get-cars` não recebe parâmetros.

Reply:
feito. https://github.com/lucas-bretzke-Warren/Everest-cars-com-Ts/pull/1/commits/9106f5e95d32558c2734e23661d963c7ec420ef6